docs/en/toolkit.md

```diff
@@ -8,6 +8,6 @@ has_children: true
 
 # Toolkit
 
-Toolkit is an extendable platform for integrating artist applications and pipeline tools with Shotgun.
+Toolkit is an extensible platform for integrating artist applications and pipeline tools with Shotgun.
 
-The Toolkit Core API is the platform upon which all integrations are built. The Core API handles app, engine, and platform fundamentals, as well as file system and deploy management.
+The Toolkit Core API is the platform upon which all integrations are built. The Core API handles app, engine, and platform fundamentals, as well as file system organization, and tool deployment.
```

Comment:
@tannazie from the "Files Changed" tab you can actually go down to the markdown itself and just click the "+" next to the line you want to comment on and the comment will be associated with that line.

Will fix.